src/Mutator/FunctionSignature/ProtectedVisibility.php

```diff
@@ -74,7 +74,7 @@ private function hasSameProtectedParentMethod(Node $node): bool
                     return true;
                 }
             } catch (\ReflectionException $e) {
-                continue;
```

Comment:
We get a reflection exception when the method doesn't exist on that class. If it doesn't exists on that class it also doesn't exist on a parent class. Since inherited, not overridden functions are also found by this. So if we get an error we know the paren't doesn't have a same protected method